src/logging.c

```diff
@@ -743,15 +743,15 @@ rcutils_ret_t rcutils_logging_format_message(
 # define SET_OUTPUT_COLOR_WITH_SEVERITY(status, severity, stream, output_array) \
   { \
     WORD color; \
```

Comment:
When the `HANDLE` variable is being complained about, should `WORD` also be default initialized then? It would make sense to me to initialize it to [COLOR_NORMAL](https://github.com/ros2/rcutils/blob/0b47fc3f2e9598e3921bf0ce1ccd0d3e5a24d400/src/logging.c#L673)

Reply:
The error in question (C4703) complains about uninitialized pointers/handles specifically. The uninitialized `color` isn't a problem. Though, say the word, and I'm happy to overload this PR and initialize it

Reply:
I would appreciate if you could initialized the color as well.

Reply:
done; sorry for delay.